simulator/setting_up_simulator.md

```diff
@@ -6,27 +6,41 @@ title: Setting up the Simulator
 Setting up the Simulator
 ========================
 
-## Prerequisites
+# Prerequisites
 
-You need to download and install [Webots](https://cyberbotics.com/#download) (the download is around 300MB).
-This is the platform we run our simulation in.
 
-Version "R2023b" of Webots is supported.
+## Webots
 
-### Python Version
+Our simulator runs in a program called Webots.
+In order to run the simulation you first need to install Webots.
+You need to download [Webots from here](https://cyberbotics.com/#download) and install it.
+
+The current supported version is "R2023b".
+
+
+## Python
 
 You will also need Python installed.
+If it is not already installed this can be downloaded and installed from the [Python website](https://www.python.org/).
 
 | Platform | Supported Python Version |
 |----------|--------------------------|
 | Windows  | 3.8-3.11 (64-bit)        |
 | macOS    | 3.8-3.11                 |
 | Linux    | 3.8-3.11                 |
 
-There are a small number of [external libraries]({{ site.baseurl }}/kit/brain_board/python_libraries) which will be available to your robot code during the competition.
-Note that for local development you will need to install these yourself.
 
-Once you have downloaded the simulation, the libraries can be installed with `pip install -r competition-simulator.../libraries.txt`.
+### Python libraries
+
+There are a small number of [external libraries]({{ site.baseurl }}/kit/brain_board/python_libraries) which are available on the physical kit that may be useful in the simulator.
+If you want to use these in your code you will need to install these yourself.
+
+Once you have downloaded the simulation, the libraries can be installed with the following command in a terminal.
+
+~~~~~bash
+pip install -r competition-simulator-<version>/libraries.txt
```

Comment:
Note: I deliberately didn't include `<version>` here as the chances of someone copy/pasting this line are quite high and they're likely to get a weird error about a file called `version` not existing (due to greater-than/less-than symbols being IO redirection in shells).

Reply:
![image](https://github.com/srobo/docs/assets/25060060/46e5427e-8dc1-493b-95c9-9e59c1980694)

Similar errors between the two on windows, I changed it so that it was consistant with all the other examples on the page.